lib/core/src/lib/card-view/validators/card-view-item-match.valiator.ts

```diff
@@ -19,16 +19,17 @@ import { CardViewItemValidator } from '../interfaces/card-view.interfaces';
 
 export interface MatchValidatorParams {
     expression: string;
+    requiresMatch?: boolean;
     flags?: string;
 }
 
 export class CardViewItemMatchValidator implements CardViewItemValidator {
     message = 'CORE.CARDVIEW.VALIDATORS.MATCH_VALIDATION_ERROR';
 
-    constructor(private expression: string, private flags?: string) {}
+    constructor(private expression: string, private requiresMatch?: boolean, private flags?: string) {}
```

Comment:
please provide unit tests to cover the newly added checks

Reply:
tests added